src/Benchmarks/Configuration/WebHostBuilderExtensions.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) .NET Foundation. All rights reserved. 
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+using System;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+
+namespace Benchmarks.Configuration
+{
+    public static class WebHostBuilderExtensions
+    {
+        private static readonly string ServerKey = "server";
```

Comment:
I would just hardcode these rather than defining constants, if they're only used a small number of times.